docs/oauth2-oidc/authorization-code-flow.mdx

```diff
@@ -6,10 +6,9 @@ toc_max_heading_level: 4
 ---
 
 OAuth2 and OpenID Connect are widely used authorization and authentication delegation protocols that enable secure access to APIs
-and web applications. In this article, we will explain the OAuth2 and OpenID Connect Authorization Code flow in a way that
-everyone can understand. We will also discuss different parameters such as `grant_type`, `redirect_uri`, `prompt`, `scope`,
-`state` and explain how to get ID tokens and refresh tokens. Additionally, we will cover how to perform PKCE on top of the
-Authorization Code flow.
+and web applications. We also discuss different parameters such as `grant_type`, `redirect_uri`, `prompt`, `scope`, `state` and
+explain how to get ID tokens and refresh tokens. Additionally, we will cover how to perform PKCE on top of the Authorization Code
+flow.
```

Comment:
This doesn't make much sense now. Can you re-write it without the "we" form and have it simply state what the doc is about?

Reply:
I just updated the text of this paragraph in the last commit.